tests/test_cli.py

```diff
@@ -1030,3 +1052,61 @@ def test_show_servers_json(
         json_output = json.loads(out)
         assert [self.test_server_name] == list(json_output.keys())
         assert json_output[self.test_server_name]["description"] == expected_description
+
+
+class TestReceiveWalCli(object):
```

Comment:
I am wondering about the need to have a dedicated class for this test. Isn't `TestCli` suitable?

Reply:
It does look a bit unnecessary doesn't it - I'll move it.